packages/jest-circus/src/run.ts

```diff
@@ -102,6 +106,10 @@ const _runTestsForDescribeBlock = async (
       // Clear errors so retries occur
       await dispatch({name: 'test_retry', test});
 
+      if (waitBeforeRetry > 0) {
+        await new Promise(resolve => setTimeout(resolve, waitBeforeRetry));
```

Comment:
does this `setTimeout` work if fake timers are active? This code runs within the sandbox of user code, so we have to for instance make sure `Date.now` is the real one: https://github.com/jestjs/jest/blob/6ead17145cd17746b7597ad8affe313683b2cd40/scripts/babel-plugin-jest-native-globals.js#L20-L22

Reply:
does using `const {setTimeout} = globalThis;` solve this?

Reply:
Yeah, that should work